src/lib.rs

```diff
@@ -35,7 +38,6 @@
 //! nine ten eleven twelve
 //! ```
 //!
-//!
 //! ## Creating a grid
 //!
 //! To add data to a grid, first create a new [`Grid`] value, and then add
```

Comment:
This sentence is outdated as there is no longer an `add` function.

And in the next sentence it should be "three options" instead of "two options".